manuscript/060_Statement_First_Programming.md

```diff
@@ -78,15 +78,15 @@ public class CalculatorSpecification
 }
 ```
 
-And you do not even notice that this will not be evaluated with the rest of the Specification, because it already consists of so many Statements that it is almost irrational to search for your added Statement in the list and make sure it is there each time. Also, note that the fact that you omitted the addition, does not disturb your work flow: test -- all pass, test -- all pass, test -- all pass... In other words, your process does not give you any feedback on your mistake. So, what you end up is a�Statement that not only will never be false -- **it will never be evaluated**.
+You will not even notice that the new Statement is not evaluated with the rest of the Specification, because that already contains so many Statements that it is unattractive to search for each new Statement in the list and make sure it's there. Also, note that the absence of the `[Fact]` attribute does not disturb your work flow: test -- all pass, test -- all pass, test -- all pass... In other words, your process does not give you any feedback that you made a mistake. So, what you end up with is a Statement that not only will never be false -- **it will never be evaluated**.
 
-How does treating tests as Statements and evaluating them before making them true help here? **Because then, a�Statement that starts off being evaluated as true is what DOES disturb your work flow.** In TDD, the work flow is: Statement -- unfulfilled -- fulfilled (ok, and refactor, but for the sake of THIS discussion, it does not matter so much), Statement -- unfulfilled -- fulfilled, Statement -- unfulfilled -- fulfilled... So every time you fail to see the �unfulfilled" stage, you get feedback from your process that something suspicious is happening. This lets you investigate and, if necessary, fix the situation at hand.
+How does treating tests as Statements and evaluating them before making them true help here? **Because then, a Statement that starts off being evaluated as true is what DOES disturb your work flow.** In TDD, the work flow is: Statement -- unfulfilled -- fulfilled (and refactor, but that doesn't matter much for this discussion), Statement -- unfulfilled -- fulfilled, Statement -- unfulfilled -- fulfilled... So every time you miss the “unfulfilled" stage, you get feedback from your process that something suspicious is happening. This lets you investigate and fix the situation if necessary.
```

Comment:
I'd change **Because then, a Statement that starts off being evaluated as true is what DOES disturb your work flow.** to just "Because then, a Statement that starts off being evaluated as true is what **does** disturb your work flow."

what was I thinking? :-D